lib/rx/client.rb

```diff
@@ -13,14 +13,18 @@ class Client < Common::Client::Base
     configuration Rx::Configuration
     client_session Rx::ClientSession
 
+    CACHE_TTL = 3600 * 10 # 10 hour cache
```

Comment:
No, i'm going to lower this to 1 hour until we can get approval for 10-24 hours.